frontend/.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules
 dist
 dist-ssr
 *.local
+package-lock.json
```

Comment:
`package-lock.json`'s should not be gitignore'd. Its absence may break the ability to start the frontend in a Docker container